extras/check_copyright_headers.py

```diff
@@ -119,22 +126,24 @@
                 try:
                     line_src = source_file.readline()
                 except UnicodeDecodeError as err:
-                    print("Unable to decode bytes in '{0}': {1}".format(tested_file, err))  # noqa
+                    print("Unable to decode bytes in '{0}': {1}".format(
+                        tested_file, err))
                     total_errors += 1
                     break
                 if (extension == 'py' and
                         line_src.strip() == '#!/usr/bin/env python'):
                     line_src = source_file.readline()
                 line_exp = template_line.replace('{{file_name}}', fname)
                 if line_src != line_exp:
-                    print("Incorrect copyright header in '{0}':".format(tested_file))           # noqa
-                    print("    expected -> '{0}', actual -> '{1}'\n".format(line_exp.strip(),   # noqa
-                                                                            line_src.strip()))  # noqa
+                    fname = os.path.relpath(tested_file)
+                    eprint("[COPY] {0}: expected '{1}', found '{2}'.".format(
```

Comment:
I do not quite understand why some things are printed with `print()`, others with `eprint()`.  Could you document this in the code?

Reply:
I've added a docstring for `eprint()` and the general working of the script in 93258e33d8.